sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -411,8 +411,12 @@ export class BatchingReceiverLite {
             // a chance to have fewer messages internally that could get lost if the user's
             // app crashes in receiveAndDelete mode.
             if (totalWaitTimer) clearTimeout(totalWaitTimer);
-
-            totalWaitTimer = setTimeout(actionAfterWaitTimeout, getRemainingWaitTimeInMs());
+            const remainingWaitTimeInMs = getRemainingWaitTimeInMs();
+            totalWaitTimer = setTimeout(() => {
+              actionAfterWaitTimeout(
```

Comment:
Now that the message is being passed to `actionAfterWaitTimeout` I'm not sure there's much value to even having `actionAfterWaitTimeout` be a separate function, but that's just a personal nit.

Reply:
I agree. @HarshaNalluru, can we inline the code from  `actionAfterWaitTimeout` here?